### DISCUSSION TITLE
Back camera is not working

### DISCUSSION THREAD

**Discussion by obidurovi** [community/community#87183] [Mobile]
### Select Topic Area

Bug

### Body

Hey, I want to register on my account for the student developer pack. I have submitted my university ID several times from my desktop. but I don't know why they are rejecting my request. However I tried to submit my ID from my mobile by taking a picture of it from GitHub's "take a photo" option but the back camera is not working, front camera is working but when I try to go for a back camera it's still showing the front camera what should I do now?

![Screenshot_20240114_120249_Chrome](https://github.com/community/community/assets/71771396/d60f224e-934f-4fff-8be3-0064e6288d21)

**Comment by anuragjain-git** [comment]
You can take the picture from your camera then upload it from choose file.

**Reply by obidurovi** [reply]
I tried but when I was doing this and i clicked "Continue". It's showing "Precessing Information...." for a long time it's loading. 

Like this ..

![Screenshot_20240114_121742_Chrome](https://github.com/community/community/assets/71771396/bfe00d70-5b93-4278-b519-e1c7f37359dd)

**Comment by anuragjain-git** [comment]
checkout #67837 same issue

**Comment by faridnec** [comment]
Hey there was a similar problem and after trying it using Desktop PC it worked. Maybe it's better to try it out using PC. Phone was problematic

**Reply by obidurovi** [reply]
Not working on pc. same bug 


![Screenshot_9](https://github.com/community/community/assets/71771396/47db62c2-b7e0-464c-a5d9-58cd147c26de)

**Reply by faridnec** [reply]
It still is processing the info, but the camera didn't work?

**Reply by obidurovi** [reply]
No, I try to upload my ID card from my desktop storage. but it's showing Precessing Information...

**Reply by faridnec** [reply]
Then I guess it means you need to wait. I also had similar case too previously

**Comment by h1ac** [comment]
I used the opera browser and It let me choose the back camera! I got accepted :)
